src/blockrelay/graphene_set.h

```diff
@@ -25,6 +25,9 @@ const float FILTER_FPR_MAX = 0.999;
 const uint8_t IBLT_CELL_MINIMUM = 2;
 const std::vector<uint8_t> IBLT_NULL_VALUE = {};
 const unsigned char WORD_BITS = 8;
+const uint16_t APPROX_NITEMS_THRESH = 600;
+const uint8_t APPROX_NEXCESS_RATE = 4;
+const float IBLT_DEFAULT_OVERHEAD = 1.5;
```

Comment:
nit:  unless there's some other reason i don't think you need the N here...just APPROX_ITEMS_THRESH and APPROX_EXCESS_RATE would be more clear

Reply:
Fixed.